tests/kvmtest.sh

```diff
@@ -320,6 +320,8 @@ export ANSIBLE_LIBRARY=$SONIC_MGMT_DIR/ansible/library/
 
 # workaround for issue https://github.com/Azure/sonic-mgmt/issues/1659
 export ANSIBLE_KEEP_REMOTE_FILES=1
+export GIT_USER_NAME=$GITHUB_USERNAME
+export GIT_API_TOKEN=$GITHUB_API_TOKEN
```

Comment:
It would be better to keep the credential consistent, like `GITHUB_USERNAME` and `GITHUB_API_TOKEN`.

Reply:
done